ee/tasks/subscriptions/slack_subscriptions.py

```diff
@@ -223,9 +223,10 @@ async def send_slack_message_with_integration_async(
 ) -> SlackDeliveryResult:
     message_data = _prepare_slack_message(subscription, assets, total_asset_count, is_new_subscription)
     slack_integration = SlackIntegration(integration)
+    async_client = slack_integration.async_client
 
     message_res = await _send_slack_message_with_retry(
-        slack_integration.async_client,
```

Comment:
doing this creates the client https://github.com/PostHog/posthog/blob/6c717af11e544af015a5b4b8ab587031effd98fc/posthog/models/integration.py#L704-L706